.env.example

```
@@ -9,18 +9,18 @@ PUBLIC_STELLAR_NETWORK=TESTNET
+PUBLIC_UNLOCK_PUBLIC_KEY=ZHVtbXktcHVibGljLWtleS12YWx1ZS1oZXJl
 
 # Optional external chat gateway used by the current UI
 PUBLIC_CHAT_API_BASE=https://secret-ai-gateway.onrender.com
 
 # Serverless unlock runtime
-CHALLENGE_TOKEN_SECRET=replace-with-a-long-random-secret
-UNLOCK_PUBLIC_KEY=BASE64_PUBLIC_KEY
-UNLOCK_PRIVATE_KEY=BASE64_PRIVATE_KEY
+CHALLENGE_TOKEN_SECRET=super-secret-challenge-token-string
+UNLOCK_PUBLIC_KEY=ZHVtbXktcHVibGljLWtleS12YWx1ZS1oZXJl
+UNLOCK_PRIVATE_KEY=ZHVtbXktcHJpdmF0ZS1rZXktdmFsdWUtaGVyZQ==
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The UNLOCK_PRIVATE_KEY key should go before the UNLOCK_PUBLIC_KEY key

(UnorderedKey)